kratos/processes/levelset_convection_process.h

```diff
@@ -840,6 +836,26 @@ class LevelSetConvectionProcess
         mEvaluateLimiter = (mIsBfecc || mElementRequiresLimiter) ? true : false;
     }
 
+    std::string GetConvectionElementRegisteredName(Parameters ThisParameters)
+    {
+        // Convection element formulation settings
+        std::string element_type = ThisParameters["element_type"].GetString();
+        const auto element_list = GetConvectionElementsList();
+        if (std::find(element_list.begin(), element_list.end(), element_type) == element_list.end()) {
+            KRATOS_INFO("") << "Specified \'" << element_type << "\' is not in the available elements list. " <<
+            "Attempting to use custom specified element." << std::endl;
+            mConvectionElementType = element_type;
+        } else {
+            mConvectionElementType = GetConvectionElementName(element_type);
+        }
+        std::string element_register_name = mConvectionElementType + std::to_string(TDim) + "D" + std::to_string(TDim + 1) + "N";
+        if (!KratosComponents<Element>::Has(element_register_name)) {
+            KRATOS_ERROR << "Specified \'" << element_type << "\' is not in the available elements list: " << element_list 
```

Comment:
But here

```suggestion
            KRATOS_INFO("") << "Specified \'" << element_type << "\' is not in the available elements list. Available elements are: \n\t- \'levelset_convection_supg\'\n\t- \'levelset_convection_algebraic_stabilization\'" <<     "\nAttempting to use custom specified element." << std::endl;
```
I mean this, so newbies (and users of the binary version) know which are the available elements without requiring to check the code.

Reply:
As pointed out by @rubenzorrilla it is essential to somehow inform the user about using the "custom" (not default) element.

Reply:
Hmm I am not sure if I am following you @rubenzorrilla . The list of available elements was being printed and I think this should be an error, as the element name is not registered in Kratos so it will throw a segmentation error later.

Reply:
I did explain my self super bad... Sorry.

Let me try again
1. We check if the element is in the elements list. If no we throw an info message saying that we will check for a custom element in the `KratosComponents`.
2. Then we do the `Has` of the `KratosComponents` as you did. If the `Has` returns `false` meaning that the custom element doesn't exist we throw the error message listing the elements i.e.
`"Specified \'" << element_type << "\' is neither in the available elements list nor registered in the KratosComponents as a custom elements. Available elements in the elements list are: \n\t- \'levelset_convection_supg\'\n\t- \'levelset_convection_algebraic_stabilization\'" << std::endl;`

(it is true that this error message is not saying which are the custom elements, but this is the best we can do I think).

I hope this clarifies.

Reply:
But this is exactly what I am currently doing :D . (See the error message in the test:)

Reply:
Only difference is that I am using the vector of element_list directly instead of explicitly writing the names. I thought this was better so if we add an element we do not have to remember to change this in the future.

Reply:
Shit. I just realized that you created an `element_list` variable. Sorry for the noise. Approving. 🤦 🤦 🤦